nilearn/regions/tests/test_region_extractor.py

```diff
@@ -173,6 +173,18 @@ def test_region_extractor_fit_and_transform():
         signal = extractor.transform(img)
         assert_equal(expected_signal_shape, signal.shape)
 
+    # smoke test with high resolution image
+    maps, mask_img = generate_maps((40, 40, 40), n_regions=n_regions,
```

Comment:
I am a bit worried that this image is big, and the test is going to take time. Is there a reason that we don't do it with an image 20x20x20?